client/app/scripts/superdesk/ui/ui.js

```diff
@@ -331,8 +331,23 @@ define([
                         css.right = Math.max(5, window.innerWidth - pos.left) + 'px';
 
                         dropdown.dropdownMenu.css({opacity: '0'}); // avoid flickering
-                        scope.$applyAsync(function() {
+
+                        scope.$applyAsync(function () {

```

Comment:
Yeah, but I need it to be executed in next $digest cycle. I need it to wait for all actions be finished, so it can calculate actual height of dropdown :) Test are comming :)